test/dotnet-compile-fsc.Tests/FSharpCompilerTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.TestFramework;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using FluentAssertions;
+using Xunit;
+
+namespace Microsoft.DotNet.Tools.Compiler.Tests
+{
+    public class FsharpCompilerTests
```

Comment:
Do you want to follow the Given name pattern? Something like: GivenThatWeWantToCompileFSharpPrograms?